src/components/pages/Dashboard/Dashboard.js

```diff
@@ -3,9 +3,10 @@ import PageHeader from '../../common/PageHeader/PageHeader';
 import StyledTable from '../../common/Table/Table';
```

Comment:
could it be possible for styledTable to have a better variable name? since its comes with export default. it can be named to suit the current use case e.g farmersTable...

Reply:
This is not part of this PR, I think this is @daquinons code. 👍

Reply:
I changed it in a PR to `DashboardTable`